beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/README.md

```diff
@@ -1,2 +1,106 @@
+
+| Task Name in WDL | Tool | Software | Description | 
+| --- | --- | --- | --- |
+| [CramToUnmappedBams](https://github.com/broadinstitute/warp/blob/develop/tasks/broad/cram_to_unmapped_bams/CramToUnmappedBams.wdl) | view, index, RevertSam, ValidateSamFile, SortSam | [SamTools](http://www.htslib.org/terms/) and [Picard](https://broadinstitute.github.io/picard/) | If a CRAM file is used as input, the task converts to uBAM and generates an output map that is then used to split the uBAM by readgroup. The resulting BAM is sorted by query name using Picard |
+| bam_readgroup_to_contents | view | [Samtools](http://www.htslib.org/terms/) | Extracts all the readgroups from the BAM header and returns a WDL array where each row is a readgroup. |
+| biobambam_bamtofastq | bamtofastq | [biobambam2](https://gitlab.com/german.tischler/biobambam2) | Converts the uBAMs to FASTQ |
+| emit_pe_records/emit_se_records | --- | --- | Associates the fasta file(s) generated by biobambam_bamtofastq with their respective readgroup; creates an array of structs to be used as input for alignment |
+| bwa_pe/ bwa_se | BWA mem, view | [BWA](http://bio-bwa.sourceforge.net/) and [Samtools](http://www.htslib.org/terms/) | Aligns FASTQ reads to the reference genome, generating an aligned BAM file |
+| picard_markduplicates | [MarkDuplicates](https://gatk.broadinstitute.org/hc/en-us/articles/360037052812-MarkDuplicates-Picard-) | [Picard](https://broadinstitute.github.io/picard/) | Runs MarkDuplicates in silent mode to locate and tag duplicate reads. Outputs a tagged BAM and metrics file. | 
+| sort_and_index_markdup_bam | sort | [Samtools](http://www.htslib.org/terms/) | Sorts and indexes the BAM file; outputs the sorted BAM and index file | 
+| gatk_baserecalibrator | [BaseRecalibrator](https://gatk.broadinstitute.org/hc/en-us/articles/360036898312) | [GATK](https://software.broadinstitute.org/gatk/download/licensing.php) | Generates recalibration table for Base Quality Score Recalibration (BQSR) | 
+| gatk_applybqsr | [ApplyBQSR](https://gatk.broadinstitute.org/hc/en-us/articles/360037055712) | [GATK](https://software.broadinstitute.org/gatk/download/licensing.php) | Recalibrates the base qualities of the input reads based on the recalibration table produced by the BaseRecalibrator tool and outputs a recalibrated BAM file; uses the emit_original_quals parameter to write the original base qualities under the BAM OQ tag |
+| collect_insert_size_metrics | [CollectInsertSizeMetrics](https://gatk.broadinstitute.org/hc/en-us/articles/360037055772) | [Picard](https://broadinstitute.github.io/picard/) | Generates metrics about insert size distribution in the form of both a histogram and table (txt file) | 
+
+## Workflow Outputs
+
+| Output Name | Description | Type |
+| --- | --- | --- |
+| validation_report | Samtools validation report(s) | txt |
```

Comment:
validation reports and unmapped bams are optional outputs that are only returned if the input is an aligned cram or bam (i.e. if CramToUnmappedBams is run)

Reply:
Updated the text to reflect the optional outputs